build.gradle

```diff
@@ -7,10 +7,3 @@ allprojects{
     apply plugin: 'nva.search.service.java-conventions'
 }
 
-tasks.withType(JavaCompile) {
```

Comment:
leftover code?

Reply:
Yes, something we tried in order for Java to compile correctly on windows, it now works correctly